### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by alexAubin** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -225,22 +93,23 @@ else
 fi
 
 # Upgrade Piwigo via cURL
-ynh_local_curl "/upgrade.php?language=$applanguage&now=true" "language=$applanguage" "username=$admin" "password=$admin_pwd"
+# FIXME : Why Curl ... Why can't we trigger the migration from the command line somehow ...
+#ynh_local_curl "/upgrade.php?language=$applanguage&now=true" "language=$applanguage" "username=$admin" "password=$admin_pwd"
```
Comment (alexAubin) [inline scripts/upgrade:97]: So tests are somewhat passing but still need to fix this .. looking at the code this doesnt seem really possible to do this from command line >_>

File: scripts/upgrade [hunk 2]
```diff
@@ -253,7 +122,9 @@ ynh_mysql_connect_as --user=$db_name --password=$db_pwd --database=$db_name <<<
 # Activate the LDAP plugin using the WS API
 
 # Login with admin account
-ynh_local_curl "/ws.php?format=json" "method=pwg.session.login" "username=$admin" "password=$admin_pwd"
+
+# FIXME: Is this stuff really needed ... isn't there a way to enable the damn LDAP login plugin from the command line or something ...
+#ynh_local_curl "/ws.php?format=json" "method=pwg.session.login" "username=$admin" "password=$admin_pwd"
```
Comment (alexAubin) [inline scripts/upgrade:127]: (also here)

**Review by alexAubin — CHANGES_REQUESTED** [review]

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade (hunk 1 above)
Comment (Salamandar) [reply scripts/upgrade:97]: What's the issue ? ynh_local_curl doesn't work anymore ?

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by alexAubin** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Pull request closed without merging**